provisioning-workflows/zOS/NetworkResources/Provision_Network_Resource.xml

```diff
@@ -0,0 +1,252 @@
+        </string>
+    </variable>
+    <variable name="OBTAIN_PORT" visibility="public">
+        <label>OBTAIN_PORT</label>
+        <abstract>Obtain Port flag</abstract>
+        <description>Flag to indicate whether or not to obtain port resource</description>
+        <category>Boolean Variables</category>
+        <string valueMustBeChoice="true">
+            <choice>YES</choice>
+            <choice>NO</choice>
+        </string>
+    </variable>
+    <variable name="OBTAIN_APPLID" visibility="public">
+        <label>OBTAIN_APPLID</label>
+        <abstract>Obtain APPLID flag</abstract>
+        <description>Flag to indicate whether or not to obtain APPLID resource</description>
+        <category>Boolean Variables</category>
+        <string valueMustBeChoice="true">
+            <choice>YES</choice>
+            <choice>NO</choice>
+        </string>
+    </variable>
+    <variable name="IP_ADDRESS" visibility="public">
+        <label>IP_ADDRESS</label>
+        <abstract>TCP/IP DVIPA IP Address</abstract>
+        <description>TCP/IP DVIPA IP Address</description>
+        <category>REST</category>
+        <string/>
+    </variable>
+    <variable name="IP_ADDRESS_ID" visibility="private">
+        <label>IP_ADDRESS_ID</label>
+        <abstract>Object ID of the IP Address resource</abstract>
+        <description>The object id of the allocated IP address to use during deprovisioning</description>
+        <category>REST</category>
+        <string/>
+    </variable>
+    <variable name="PORT" visibility="public">
+        <label>PORT</label>
+        <abstract>TCP/IP Port Number</abstract>
+        <description>TCP/IP port number</description>
+        <category>REST</category>
+        <string/>
+    </variable>
+    <variable name="PORT_ID" visibility="private">
+        <label>PORT_ID</label>
+        <abstract>Object ID of the port resource</abstract>
+        <description>The object id of the reserved Port address to use during deprovisioning</description>
+        <category>REST</category>
+        <string/>
+    </variable>
+    <variable name="APPLID" visibility="public">
+        <label>APPLID</label>
+        <abstract>VTAM APPLID</abstract>
+        <description>VTAM APPLID</description>
+        <category>REST</category>
+        <string/>
+    </variable>
+    <variable name="APPLID_ID" visibility="private">
+        <label>APPLID_ID</label>
+        <abstract>Object ID of the APPLID resource</abstract>
+        <description>The object id of the reserved APPLID resource to use during deprovisioning</description>
+        <category>REST</category>
+        <string/>
+    </variable>
+    <variable name="JOB_NAME" visibility="public">
+        <label>JOB_NAME</label>
+        <abstract>Job name of server that will use allocated port. Specify * if Jobname is not available</abstract>
+        <description>Job name of server that will use allocated port. Specify * if Jobname is not available</description>
```

Comment:
Is this a question of availability, or more of a question as to if the user knows a specific job name to use?

Reply:
Both.. In some cases e.g. when application runs as unix process, the jobname is internally generated and not available/known to user. I will change "available" to "known" to avoid any confusion